### PR TITLE
BaSP-TG: superAdmin - added post method

### PR DESCRIPTION
**Pull request by silveriac**

### CODE REVIEW

**Review by joryapur — COMMENTED** [review]
I think everything is very good, clear and concise. The only thing that may be missing is the last name in the json. Good job mate

File: src/resources/super-admins.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const express = require("express")
+const superAdmins = require('../data/super-admins.json');
+const fs = require('fs');
+const router = express.Router();
+router.use(express.urlencoded({extended: false}));
```
Comment (nicolasblasco) [inline src/resources/super-admins.js:5]: We should move this to index.js

File: src/resources/super-admins.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+router.post('/post/', (req, res) => {
+    let newSuperAdmin = req.body;
+    newSuperAdmin.id = (superAdmins.length + 1).toString();
+    superAdmins.push(newSuperAdmin);
+
+    fs.writeFile("./src/data/super-admins.json", JSON.stringify(superAdmins, null, 2), (err) => {
+        if (err) {
+            console.log(err);
```
Comment (nicolasblasco) [inline src/resources/super-admins.js:14]: We should remove this console.log

File: src/resources/super-admins.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        };
+        res.status(201).json({
+            success: true,
+            msg: "Super Admin created successfully",
+            data: newSuperAdmin
+        });
+    });
+});
+
+module.exports = router;
```
Comment (nicolasblasco) [inline src/resources/super-admins.js:27]: When using import this should change to export default

**Review by nicolasblasco — APPROVED** [review]

**Pull request merged**